Makefile

```diff
@@ -27,10 +31,30 @@ patch:
 	find ${SPEC_PATCHED_FILE} -empty -exec cp ${SPEC_FETCHED_FILE} ${SPEC_PATCHED_FILE} \;
 
 gen:
-	${SWAGGER} generate client \
+	${SWAGGER} generate -g go \
+		--package-name ${PACKAGE_MAJOR} \
 		--model-package types \
-		--additional-initialism bgp \
-		--additional-initialism vpn \
-		--additional-initialism vlan \
-		--additional-initialism vlans \
-		-f ${SPEC_PATCHED_FILE}
+		--api-package models \
+		--git-user-id ${GIT_ORG} \
+		--git-repo-id ${GIT_REPO} \
+		-o /local/${PACKAGE_MAJOR} \
+		-i /local/${SPEC_PATCHED_FILE}
+
+# https://github.com/OpenAPITools/openapi-generator/issues/741#issuecomment-569791780
+remove-dupe-requests: ## Removes duplicate Request structs from the generated code
```

Comment:
This can be removed, not actually being used and unneeded after the extra tags were removed from each resource.